build-linux.sh

```diff
@@ -3,11 +3,24 @@ set -e
 
 export PATH="$HOME/miniconda/bin:$PATH"
 
+# add flat buffers
+cd ..
+git clone https://github.com/google/flatbuffers.git
+cd ./flatbuffers
+git checkout 925c1d77fcc72636924c3c13428a34180c30f96f  # release commit
+cmake .
+make
+sudo make install
+sudo cp ./flatc /usr/local/bin/
```

Comment:
Does the install target not install the executable?

Reply:
no, it won't.